client/js/store/modules/game.js

```diff
@@ -277,26 +277,26 @@ export default {
      */
     processScuttle(context, { game, playedCardId, targetCardId, playedBy }) {
       // Update in one step if this player scuttled or if pNum is not set
-      if (!context.getters.player || playedBy === context.state.myPNum) {
+      if (!context.getters.player) {
         context.dispatch('updateGameThenResetPNumIfNull', game);
         return;
       }
-      // Remove played card from opponent hand and temporarily add to its targets attachments
-      const playedCardIndex = context.getters.opponent.hand.findIndex(
-        (card) => card.id === playedCardId
-      );
-      const targetCardIndex = context.getters.player.points.findIndex(
-        (card) => card.id === targetCardId
-      );
+
+      const scuttlingPlayer = context.state.players[playedBy];
+      const scuttledPlayer = context.state.players[(playedBy + 1) % 2];
```

Comment:
`playedBy` is the `pNum` of which player is scuttling ie the index in the array of players, so this grabs the player objects so that we can remove the scuttling card from the scuttling player's hand and add it to the target cards inside the scuttled player's points